components/sidebar/sidebar_service.cc

```diff
@@ -603,7 +604,8 @@ SidebarItem SidebarService::GetBuiltInItemForType(
     }
     case SidebarItem::BuiltInItemType::kPlaylist: {
       if (base::FeatureList::IsEnabled(playlist::features::kPlaylist)) {
-        return SidebarItem::Create(brave_l10n::GetLocalizedResourceUTF16String(
+        return SidebarItem::Create(GURL(kPlaylistURL),
```

Comment:
Just curious - Is there any specific reason to pass url because passed url here is not used?
I'm fine if it's for just filling data.

Reply:
Yep, not used but for filling data :)